databases/week3/Additional_queries.sql

```diff
@@ -0,0 +1,44 @@
+-- Additional Quries
+-- Get meals that has a price smaller than a specific price fx 90
+select * from meal where price <90;
+
+-- Get meals that partially match a title
+select * from meal where title like '%butter%'; 
+
+-- Get meals that still has available reservations
+SELECT meal.id as meal_id, title, discription, location, price, max_reservations, 
+	COALESCE(SUM(number_of_guests), 0) AS total_reservations, 
+    COALESCE((max_reservations - SUM(number_of_guests)), max_reservations) as available_spots
+FROM meal
+LEFT JOIN reservation
+	ON meal.id = meal_id
+GROUP BY meal.id
+HAVING  max_reservations > total_reservations is null
```

Comment:
```suggestion
HAVING  max_reservations > total_reservations
```

I think you forgot to remove the `is null` part :)